domains/burmeselofi.localplayer.dev.json

```diff
@@ -0,0 +1,13 @@
+{
+    "description": "Music Download Site",
+    "domain": "localplayer.dev",
+    "subdomain": "*burmeselofi",
```

Comment:
If you are wanting to use a wildcard for your subdomain you need to make it `*.burmeselofi` so then it will work, but you also need to give is a reason as to why you need a **wildcard** for your subdomain.